### DISCUSSION TITLE
Weird behaviour when trying to import

### DISCUSSION THREAD

**Discussion by jyrg02** [beetbox/beets#4555] [Q&A] [answered]
CMD Input 
![image](https://user-images.githubusercontent.com/99687156/202594937-6c69193b-c6f8-4474-a611-af312fc6fe86.png)
Is this supposed to happen?

**Comment by jyrg02** [comment]
Follow up: 
It does the same with other stuff. Here's the raw code for the other one: 

`C:\Windows\System32>beet import C:\Users\Lenovo\Music
Import of the directory:
C:\Users\Lenovo\Music
was interrupted. Resume (Y/n)? Y
Resuming interrupted import of C:\Users\Lenovo\Music
Skipped 6 paths.

C:\Users\Lenovo\Music\Sheila Fleming-20220306T135408Z-001 (12 items)
Finding tags for album "Sheila Fleming - Circles".
Candidates:
1. Thorn.Eleven - Circles (←[31;01m47.9%←[39;49;00m) ←[33;01m(tracks, artist, year)←[39;49;00m ←[37m(CD, 2009, DE, Muscon Records)←[39;49;00m
2. News for Lulu - Circles (←[31;01m46.9%←[39;49;00m) ←[33;01m(tracks, artist, year)←[39;49;00m ←[37m(CD, 2014, IT, Urtovox)←[39;49;00m
3. Amanaska - Circles (←[31;01m39.3%←[39;49;00m) ←[33;01m(tracks, artist, year)←[39;49;00m ←[37m(CD, 2006, AU, One World Music, OWM020)←[39;49;00m
4. The New Seekers - Circles (←[31;01m39.2%←[39;49;00m) ←[33;01m(tracks, artist, year)←[39;49;00m ←[37m(12" Vinyl, 1972, US, Elektra, EKS-75034)←[39;49;00m
5. Assassination - Circles Within Circles (←[31;01m34.8%←[39;49;00m) ←[33;01m(tracks, artist, album, ...)←[39;49;00m ←[37m(CD, 2019, DE, [none])←[39;49;00m
# selection (default ←[36;01m1←[39;49;00m), ←[34;01mS←[39;49;00mkip, ←[34;01mU←[39;49;00mse as-is, as ←[34;01mT←[39;49;00mracks, ←[34;01mG←[39;49;00mroup albums,
←[34;01mE←[39;49;00mnter search, enter ←[34;01mI←[39;49;00md, a←[34;01mB←[39;49;00mort?
Correcting tags from:
    ←[31;01mS←[39;49;00mh←[31;01meila F←[39;49;00mle←[31;01mmi←[39;49;00mn←[31;01mg←[39;49;00m - Circles
To:
    ←[31;01mT←[39;49;00mh←[31;01morn.E←[39;49;00mle←[31;01mve←[39;49;00mn - Circles
URL:
    https://musicbrainz.org/release/0638a1d9-e1a6-4416-8b2b-0b58bbb652fa
(Similarity: ←[31;01m47.9%←[39;49;00m) ←[33;01m(tracks, artist, year)←[39;49;00m ←[37m(CD, 2009, DE, Muscon Records)←[39;49;00m
 * ←[31;01mCi←[39;49;00mr←[31;01mcles←[39;49;00m←[31;01m (4:40)←[39;49;00m ->
   ←[31;01mInt←[39;49;00mr←[31;01mo←[39;49;00m←[31;01m (1:33)←[39;49;00m ←[33;01m(title, length)←[39;49;00m
 * ←[31;01mNot A Chr←[39;49;00mis←[31;01mtm←[39;49;00ma←[31;01ms So←[39;49;00mn←[31;01mg←[39;49;00m←[31;01m (#8)←[39;49;00m←[31;01m (6:58)←[39;49;00m ->
   ←[31;01mQu←[39;49;00mi←[31;01mck←[39;49;00msan←[31;01md←[39;49;00m←[31;01m (#2)←[39;49;00m←[31;01m (3:02)←[39;49;00m ←[33;01m(title, length, index)←[39;49;00m
 * ←[31;01mLik←[39;49;00me←[31;01m A Fi←[39;49;00mr←[31;01me←[39;49;00m←[31;01m (4:42)←[39;49;00m ->
   ←[31;01mSumm←[39;49;00mer←[31;01m (3:08)←[39;49;00m ←[33;01m(title, length)←[39;49;00m
 * ←[31;01mBetween My L←[39;49;00mi←[31;01mn←[39;49;00mes ->
   ←[31;01mC←[39;49;00mi←[31;01mrcl←[39;49;00mes ←[33;01m(title)←[39;49;00m
 * ←[31;01mL←[39;49;00ma←[31;01my Your Burd←[39;49;00me←[31;01mns Down←[39;49;00m ->
   ←[31;01mMir←[39;49;00ma←[31;01mcl←[39;49;00me ←[33;01m(title)←[39;49;00m
 * Lo←[31;01mve I←[39;49;00ms←[31;01mn'←[39;49;00mt←[31;01m Dead←[39;49;00m←[31;01m (#12)←[39;49;00m←[31;01m (4:27)←[39;49;00m ->
   Lost←[31;01m (#6)←[39;49;00m←[31;01m (4:40)←[39;49;00m ←[33;01m(title, index, length)←[39;49;00m
 * ←[31;01mTh←[39;49;00me←[31;01m Hero's Stor←[39;49;00my←[31;01m (5:10)←[39;49;00m ->
   ←[31;01mSom←[39;49;00me←[31;01mda←[39;49;00my←[31;01m (3:30)←[39;49;00m ←[33;01m(title, length)←[39;49;00m
 * ←[31;01mV←[39;49;00mery←[31;01m Small Town←[39;49;00m←[31;01m (#9)←[39;49;00m←[31;01m (3:22)←[39;49;00m ->
   ←[31;01mMis←[39;49;00mery←[31;01m (#8)←[39;49;00m←[31;01m (3:12)←[39;49;00m ←[33;01m(title, index, length)←[39;49;00m
 * ←[31;01mWhat It's Not←[39;49;00m←[31;01m (#6)←[39;49;00m←[31;01m (3:45)←[39;49;00m ->
   ←[31;01mAidin←[39;49;00m←[31;01m (#9)←[39;49;00m←[31;01m (3:56)←[39;49;00m ←[33;01m(title, index, length)←[39;49;00m
 * ←[31;01mThe Turning Of A Phrase←[39;49;00m←[31;01m (4:05)←[39;49;00m ->
   ←[31;01mSolo←[39;49;00m←[31;01m (3:51)←[39;49;00m ←[33;01m(title, length)←[39;49;00m
 * ←[31;01mTime I←[39;49;00mn←[31;01m The Pas←[39;49;00mt←[31;01m (3:26)←[39;49;00m ->
   ←[31;01mCo←[39;49;00mnt←[31;01mrol←[39;49;00m←[31;01m (3:44)←[39;49;00m ←[33;01m(title, length)←[39;49;00m
 * ←[31;01mOde T←[39;49;00mo ←[31;01mA←[39;49;00m ←[31;01mWoolworth'←[39;49;00ms ←[31;01mStor←[39;49;00me←[31;01m (#2)←[39;49;00m←[31;01m (5:33)←[39;49;00m ->
   ←[31;01mD←[39;49;00mo ←[31;01mYou←[39;49;00m ←[31;01mMis←[39;49;00ms ←[31;01mM←[39;49;00me←[31;01m (#12)←[39;49;00m←[31;01m (5:46)←[39;49;00m ←[33;01m(title, index, length)←[39;49;00m
←[34;01mA←[39;49;00mpply, ←[34;01mM←[39;49;00more candidates, ←[34;01mS←[39;49;00mkip, ←[34;01mU←[39;49;00mse as-is, as ←[34;01mT←[39;49;00mracks, ←[34;01mG←[39;49;00mroup albums,
←[34;01mE←[39;49;00mnter search, enter ←[34;01mI←[39;49;00md, a←[34;01mB←[39;49;00mort?
Enter one of A, M, S, U, T, G, E, I, B: B`

**Comment by sampsyo** [comment, marked as answer]
Please see #2607.
